app/services/loggingService/loggingService.service.spec.js

```diff
@@ -1,9 +1,52 @@
-'use strict';
-describe('loggingService', function () {
+"use strict";
 
-  it('TBD',function() {
+describe('featureToggleService', function () {
+
+  var lodash = _;
+
+  beforeEach(module('myApp.services', function($provide){
+  	$provide.value('API_URLS', 'config-featureToggle-link');
+  }));
+
+  it('should verify default toggles and config',inject(function(featureToggleService) {
+  	expect(featureToggleService.getToggles()).toEqual({});
+  	expect(featureToggleService.getConfig()).toEqual({
+          enableDates: true, 
+          hosts: {
+            localhost: {
+              enableAll: false,
+              disableAll: false
+            }
+          }
+        });
+  }));
+
+it('should be disable for any path default',inject(function(featureToggleService) {
+  	expect(featureToggleService.isEnabled('rest.serviceA')).toBe(false);
+  }));
+
```

Comment:
spyOn($location.host).and.returnValue('localhost');  is not necessary here. does nothing for test.

describe is misconstrued.  expect returns false because toggle path is not defined.